Documentation/using-nlog.md

```diff
@@ -0,0 +1,134 @@
+# Using NLog
```

Comment:
This should be called Logging guidelines or whatever. 
You can also ignore the fact that we 'moved' to Nlog, maybe just say that we use NLog to help us with the logging.